front-end/src/components/RollCallComponents/CallsComponent.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<div class="flex flex-col items-center mt-10">
+		<qrcode v-if="url" :value="url" />
+		<div v-else>
+			<p>{{ message }}</p>
+		</div>
+		<a :href="url" v-if="url" target="_blank">
+			<button class="bg-gray-500 text-black py-0.5 px-5 rounded mt-2 mb-4">Ouvrir le lien</button>
+		</a>
+	</div>
+</template>
+
+
+<script lang="ts">
+import Qrcode from 'vue-qrcode';
+import { http } from '@/api/network/axios';
+import { withErrorHandler } from '@/utils/storeHandler';
+
+let url = '';
```

Comment:
Why are these defined both in the data and outside the component as global var ?